src/base/bittorrent/torrentimpl.cpp

```diff
@@ -78,6 +78,16 @@ namespace
         return entry;
     }
 
+    QDateTime fromLTTimePoint32(const lt::time_point32 &timePoint)
+    {
+        using namespace std::chrono;
+
+        const auto ltNow = lt::clock_type::now();
+        const auto secsSinceNow = duration_cast<seconds>(timePoint - ltNow + lt::milliseconds(500)).count();
+        const auto qNow = QDateTime::currentDateTime();
+        return qNow.addSecs(secsSinceNow);
```

Comment:
Still, I think you should stick to the same namespace even if the underlying is the same.
```suggestion
        const auto ltNow = lt::clock_type::now();
        const auto qNow = QDateTime::currentDateTime();  // moved to minimize time difference
        const auto secsSinceNow = lt::duration_cast<lt::seconds>(timePoint - ltNow + lt::milliseconds(500)).count();
        return qNow.addSecs(secsSinceNow);
```

Reply:
> Still, I think you should stick to the same namespace even if the underlying is the same.

Ok. Done.